### DISCUSSION TITLE
Server abort in docker

### DISCUSSION THREAD

**Discussion by karbobc** [dani-garcia/vaultwarden#2779] [Q&A]
### Subject of the issue

Server will be aborted after the docker running 3-4 days.

### Deployment environment

* vaultwarden version:
1.25.2

* Install method:
Docker image

* Clients used: 
Web vault

* Reverse proxy and version: 
Nginx 1.20.2

* MySQL/MariaDB or PostgreSQL version: 
PostgreSQL 14.5

### Troubleshooting data

```log
crypto/evp/evp_enc.c:512: OpenSSL internal error: assertion failed: b <= sizeof(ctx->final)
```

**Comment by BlackDex** [comment]
Do you have SSL configured on the Vaultwarden side also?
And what are the error message surrounding this message?

Because, with only this message we can't do anything.
Also, see: #2393 which has the exact same message.

**Reply by karbobc** [reply]
I didn't configure SSL at Vaultwarden, use nginx reverse proxy instead.
Only one error message in the log file.

**Reply by karbobc** [reply]
I'm trying to use `sslmode=disable` in PostgreSQL URI.

**Reply by BlackDex** [reply]
Could you try the Alpine based image and see if that solves the issue?
Release: `vaultwarden/server:alpine`
Testing: `vaultwarden/server:testing-alpine`

**Reply by karbobc** [reply]
`vaultwarden/server:alpine` is my current docker image.

**Reply by BlackDex** [reply]
And what if you try `testing-alpine`? the release version is based upon 3.15.5, the testing is 3.16.2
